lldb/test/Shell/Settings/TestEchoCommands.test

```diff
@@ -2,6 +2,8 @@
 # RUN: %lldb -x -b -o 'settings set interpreter.echo-comment-commands false' -s %S/Inputs/EchoCommandsTest.in | FileCheck %S/Inputs/EchoCommandsNoComments.out
 # RUN: %lldb -x -b -o 'settings set interpreter.echo-commands false'         -s %S/Inputs/EchoCommandsTest.in | FileCheck %S/Inputs/EchoCommandsNone.out
 
+XFAIL: remote{{.*}}
```

Comment:
Why exactly does this fail when run remote? I get that in some sense it doesn't apply because nothing would happen on the remote, but I'm surprised that it fails.

Is it because with a remote, %lldb is actually `lldb -o platform connect` etc and that screws up the output?

Reply:
I would add the explanation as a comment, even if it is obvious in hindsight.

Reply:
Yes, you're right.
However, the only test case that fails here is "quiet", so I've committed a more precise version to this PR, where TestEchoCommands.test is split into TestEchoCommandsQuiet.test (quiet test case) and TestEchoCommands.test (the rest), and where the only disabled part is TestEchoCommandsQuiet.test.